pkg/server/server.go

```diff
@@ -53,6 +52,22 @@ const (
 	grpcDialTimeout = 5 * time.Second
 )
 
+func ServeUI(ctx context.Context, logger log.Logger, cfg UIConfig) {
+	isUIPortNotExits := cfg.Port == 0
+	if isUIPortNotExits {
+		logger.Error("ui server failed: no port specified")
+		return
+	}
+
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		http.ServeFile(w, r, "ui/dist/ui/index.html")
+	})
+
+	logger.Info("ui server starting", "http-port", cfg.Port)
+	utils.Open(fmt.Sprintf("http://localhost:%d", cfg.Port))
```

Comment:
why do we need to open UI in browser, can we have config to disable this?

Reply:
Yeah we should remove this